app/views/scenarios/slides/_interconnector_curves_upload.html.haml

```diff
@@ -0,0 +1,9 @@
+.interface_group
+  %hr
+  %span.label= t("custom_curves.interface_group_title")
+
+%p{ style: 'margin-top: 0' }
+  = t("custom_curves.interconnector_intro_html", docs_path: 'https://docs.energytransitionmodel.com/main/costs-imported-electricity#uploaded-price-curves')
```

Comment:
@antw is it possible to make this link open in a new tab? When I click it now I'm immediately referred to the documentation, thereby leaving the model, while we normally let external links in the model open in a new tab.

Reply:
Done.